pkg/config/config.go

```diff
@@ -145,3 +151,19 @@ func (c *HarvesterConfig) String() string {
 	}
 	return fmt.Sprintf("%+v", *s)
 }
+
+func (c *HarvesterConfig) GetKubeletArgs() []string {
+	// node-labels=key1=val1,key2=val2
+	labelStrs := make([]string, 0, len(c.Labels))
+	for key, val := range c.Labels {
+		labelStrs = append(labelStrs, fmt.Sprintf("%s=%s", key, val))
```

Comment:
Can we add some validation? e.g., `k8s.io/apimachinery/pkg/util/validation.IsQualifiedName/IsValidLabelValue`

Reply:
Good idea. Will add this validation.